### DISCUSSION TITLE
Syncthing

### DISCUSSION THREAD

**Discussion by dibu28** [tteck/Proxmox#373] [Ideas]
Hello.
Can you please add Syncthing as LXC container?
One line setup script.

**Comment by tteck** [comment]
Done!

**Comment by Fabick75** [comment]
Just a question how could update the software version inside the container, if there are updates? Any procedure suggested, or we need to destroy and recreate the container? In that case how to preserve the configurations ?

**Reply by tteck** [reply]
`apt-get update && apt-get upgrade`
